pkg/inject/sidecar_test.go

```diff
@@ -474,3 +498,42 @@ func containsOptionWithPrefix(t *testing.T, args []string, prefix string) bool {
 	assert.Fail(t, "list of arguments didn't have an option starting with '%s'", prefix)
 	return false
 }
+
+func containsAgentTag(t *testing.T, args []string, tag string, value string) bool {
```

Comment:
We have a `util.FindItem(prefix string, args []string)` already that you could make use of to get the agent tag. With that, it's probably sufficient to use something like `assert.Contains(t, tag, value)`.